worlds/timespinner/Locations.py

```diff
@@ -18,257 +18,257 @@ def get_locations(world: Optional[MultiWorld], player: Optional[int]) -> Tuple[L
+        LocationData('Sealed Caves (Xarion)', 'Sealed Caves: bottom left room',  1337050),
+        LocationData('Sealed Caves (Xarion)', 'Sealed Caves: Last chance before Xarion',  1337051, lambda state: state._timespinner_has_doublejump(world, player)),
+        LocationData('Sealed Caves (Xarion)', 'Sealed Caves: Xarion',  1337052),
+        LocationData('Sealed Caves (Sirens)', 'Sealed Caves: Upper water hook',  1337053, lambda state: state.has('Water Mask', player)),
+        LocationData('Sealed Caves (Sirens)', 'Sealed Caves: Upper siren room underwater right',  1337054, lambda state: state.has('Water Mask', player)),
+        LocationData('Sealed Caves (Sirens)', 'Sealed Caves: Upper siren room underwater left',  1337055, lambda state: state.has('Water Mask', player)),
+        LocationData('Sealed Caves (Sirens)', 'Sealed Caves: Upper cave after sirens chest 1',  1337056),
+        LocationData('Sealed Caves (Sirens)', 'Sealed Caves: Upper cave after sirens chest 2',  1337057),
+        LocationData('Military Fortress', 'Military Fortress: Bomber chest',  1337058, lambda state: state.has('Timespinner Wheel', player) and state._timespinner_has_doublejump_of_npc(world, player)),
+        LocationData('Military Fortress', 'Military Fortress: Close combat room',  1337059),
+        LocationData('Military Fortress (hangar)', 'Military Fortress: Soldiers bridge',  1337060),
+        LocationData('Military Fortress (hangar)', 'Military Fortress: Giantess room',  1337061),
+        LocationData('Military Fortress (hangar)', 'Military Fortress: Giantess bridge',  1337062),
+        LocationData('Military Fortress (hangar)', 'Military Fortress: B door chest 2',  1337063, lambda state: state._timespinner_has_doublejump(world, player) and state._timespinner_has_keycard_B(world, player)),
+        LocationData('Military Fortress (hangar)', 'Military Fortress: B door chest 1',  1337064, lambda state: state._timespinner_has_doublejump(world, player) and state._timespinner_has_keycard_B(world, player)),
+        LocationData('Military Fortress (hangar)', 'Military Fortress: Pedestal',  1337065, lambda state: state._timespinner_has_doublejump_of_npc(world, player) or state._timespinner_has_forwarddash_doublejump(world, player)),
+        LocationData('The lab', 'Lab: Coffee break',  1337066),
+        LocationData('The lab', 'Lab: Lower trash right',  1337067, lambda state: state._timespinner_has_doublejump(world, player)),
+        LocationData('The lab', 'Lab: Lower trash left',  1337068, lambda state: state._timespinner_has_upwarddash(world, player)),
+        LocationData('The lab', 'Lab: Below lab entrance',  1337069, lambda state: state._timespinner_has_doublejump(world, player)),
+        LocationData('The lab (power off)', 'Lab: Trash jump room',  1337070),
+        LocationData('The lab (power off)', 'Lab: Dynamo Works',  1337071),
+        LocationData('The lab (upper)', 'Lab: Genza',  1337072),
```

Comment:
Blob mom is here to stay as a fan favourite

Reply:
I knew I would get pushback on this. I mainly did this to remove some of the more story spoiler-y names in some of the locations, which is why I also removed all references to dad. I know blob mom is a favorite though I would like to hear from more of the community before either commiting or reverting this change.

Reply:
if you're playing the rando and haven't played the vanilla game that's entirely the fault of the player and i see no reason to keep spoilery content out of a rando that's intended to change up the gameplay experience.

Reply:
imo nothing is stopping us from referring to these checks as their OG names. Like I'm still gonna call it daddy's bottom regardless of what it's called in a file I'm never going to look at, lol

Reply:
yes but then you have a split in community of not knowing the names. it's like lttp's "hype cave" that everyone agrees shouldn't really be called that anymore but changin the name would just add confusion at this point as people would still refer to it that way.